lib/logidze/versioned_association.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+module Logidze
+  module VersionedAssociation
+    def load_target
+      target = super
+
+      return target if inversed
+
+      time = owner.log_data.current_version.time
+
+      if target.is_a? Array
+        target.map! do |object|
+          object unless has_logidze? object
+
+          object.at(time)
```

Comment:
It's strange, but when I use `at`, test case with empty comments failed. Somehow `super` return `[false]` instead of `[]`, so `has_logidze?` fails. I guess, somehow `at!` messes up something inside. I can investigate it, if it's important